wopiserver/values.yaml

```diff
@@ -39,49 +40,6 @@ ingress:
     #   hosts:
     #     - wopi.local
 
-wopibridge:
-  enabled: false
-  name: wopibridge
-  replicaCount: 1
-  image:
-    repository: cs3org/wopibridge
-    tag: v4.0.0
-    pullPolicy: Always
-
-  service:
-    type: ClusterIP
-    port: 8000
-
-  ## Overrides the default URL generation based on the ingress/service values
-  # bridgeUrl: https://bridge.local/wopibridge
-
-  extraVolumes: []
-    # - name: image-store
-    #   persistentVolumeClaim:
-    #     claimName: image-store
-
-  extraVolumeMounts: []
-    # - name: image-store
-    #   mountPath: /mnt/codimd_storage
-
-  env:
-    CODIMD_INT_URL: http://codimd
-    CODIMD_EXT_URL: https://codimd.local/path
-
   ## Sets a pre-defined secrets instead of auto-generating them
   secrets: {}
-    # wbsecret: myWopiBridgeSecret
     # codimd_apikey: codimdSharedSecret
-
-  ingress:
```

Comment:
Is exposing the bridge's URL no longer required? Otherwise, we should move these rules to the wopiserver ingress template.

Reply:
Good question (and thanks for all other comments, will address them): it's no longer required, assuming that the apps (CodiMD) run in the same K8s cluster. If/when this is not the case, and apps are deployed externally, then we will need to expose URLs of the form `https://<wopiserver>/wopi`, and specifically for CodiMD it's `https://<wopiserver>/wopi/bridge/<docid>` (so comprised in the general wopiserver URL).